.gitlab-ci.yml

```diff
@@ -114,13 +114,22 @@ default:
     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
 
 .build-refs:                       &build-refs
+  variables:
```

Comment:
please do not add `variables:` into a `rules:` anchor. If the job uses this anchor and has `variables:` they will conflict.

Reply:
interesting, anyway noticed I need to edit this section because it was triggering the build of  sub-key also :)

Reply:
should be handled